internal/contentcache/contentcache.go

```diff
@@ -152,11 +151,19 @@ func (c *ContentCache) RecoverCache() error {
 		c.tempDir = "/tmp"
 	}
 	logger.Infof("Recovering cache:\n")
-	files, err := ioutil.ReadDir(c.tempDir)
+	dirEntries, err := os.ReadDir(c.tempDir)
 	if err != nil {
 		// if we fail to read the specified directory, log and return error
 		return fmt.Errorf("recover cache: %w", err)
 	}
+	files := make([]os.FileInfo, len(dirEntries))
+	for i, dirEntry := range dirEntries {
+		files[i], err = dirEntry.Info()
+		if err != nil {
+			// if we fail to read the specified directory, log and return error
```

Comment:
you need to update the comment here. We failed to read the specific directory entry.

Reply:
Done.